wikichange/src/content/root.js

```diff
@@ -0,0 +1,12 @@
+import { text } from "./tagEveryWord";
+import { markPage, removeMarks } from "./markPage";
+import "./content";
+
+console.info(text.substring(0, 21));
```

Comment:
Let's move this code to the readme, as it is test code and not actually "prod ready"

Reply:
Eg: ### How to tag every word